docs/guides/intro-to-graphql.md

```diff
@@ -1,9 +1,407 @@
+
+#### Usage
+
+```gql
+mutation {
+  updateUser(id: ID, data: { name: "Simon" }) {
+    id
+  }
+}
+```
+
+### updateUsers
+
+Update multiple Users by ID. Accepts a single data parameter that contains an array of objects. The object parameters are the same as `createUser` and should contain an `id` and nested `data` parameter with the field data.
+
+```gql
+mutation {
+  updateUsers(data: [{ id: ID, data: { name: "Simon" } }]) {
+    id
+  }
+}
+```
+
+### deleteUser
+
+Delete a single Entity by ID. Accepts a single parameter where the `id` matches a user id.
+
+```gql
+mutation {
+  deleteUser(id: ID)
+}
+```
+
+### deleteUsers
+
+Delete multiple entities by ID. Similar to `deleteUser` where the `id` parameter is an array of ids.
+
+```gql
+mutation {
+  deleteUsers(id: [ID])
+}
+```
+
+## Executing Queries and Mutations
+
+Before you begin writing application code, a great place test queries and mutations is the [GraphQL Playground](https://www.apollographql.com/docs/apollo-server/features/graphql-playground/). The default path for Keystone's GraphQl Playground is `http://localhost:3000/admin/graphql`. Here you can execute queries and mutations against the Keystone API without writing any JavaScript.
+
+Once you have determined the correct query or mutation, you can add this to a front-end application. To do this you will need to submit a `POST` request to Keystone's API. The default API endpoint is: `http://localhost:3000/admin/api`.
+
+In our examples we're going to use the browser's [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) for older browsers you may need to polyfill this or use a library like [axios](https://github.com/axios/axios).
+
+First of all let's save a simple query in a variable:
+
+```javascript
+const GET_ALL_USERS = `
+query GetUsers {
+  allUsers {
+    name
+    id
+  }
+}`;
+```
+
+We can then execute this query using a `POST` request:
+
+```javascript
+fetch('/admin/api', {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+  },
+  body: {
+    query: GET_ALL_USERS,
+  },
+}).then(result => result.json());
+```
+
+The result should contain a `JSON` payload with the results from the query.
+
+Executing mutations is the same, however we need to pass variables along with the mutation:
+
+```javascript
+const ADD_USER = `
+mutation AddUser($name: String!) {
+  createUser(data: { name: $name }) {
+    name
+    id
+  }
+}`;
+```
+
+The key for mutations is still `query`:
+
+```javascript
+fetch('/admin/api', {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+  },
+  body: {
+    query: ADD_USER,
+    variables: { name: 'Mike' },
+  },
+}).then(result => result.json());
+```
+
+**Note: ** Queries via the API will enforce [Access Control](https://v5.keystonejs.com/api/access-control).
```

Comment:
While important, is this an appropriate point to bring up Access Control? It's a potential rabbit hole and throws off the cadence of this guide.